io.github.theforceengine.tfe.yml

```diff
@@ -0,0 +1,69 @@
+app-id: io.github.theforceengine.tfe
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: io.github.theforceengine.tfe.sh
+
+finish-args:
+  # hardware 3D
+  - --device=dri
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Audio
+  - --socket=pulseaudio
+  # Access to the default Steam Dark Forces data directory
+  - --filesystem=~/.local/share/Steam/steamapps/common/Dark Forces/Game/
+
+modules:
+  - shared-modules/glu/glu-9.json
+  - shared-modules/glew/glew.json
+
+  - name: tfe
+    sources:
+      - type: git
+        url: https://github.com/luciusDXL/TheForceEngine.git
+        tag: v1.09.540
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\\d.]+)$
+      # Separate engine data from user data on Linux by setting PATH_USER_DOCUMENTS to XDG_HOME.
+      - type: patch
+        path: user_data.patch
+      # Patch automatic detection of Dark Forces installation to only check default Steam path
+      - type: patch
+        path: game_data_detection.patch
+      # Include local appdata.xml file until a TFE release is made that includes it
+      # https://github.com/luciusDXL/TheForceEngine/pull/375
+      - type: file
+        path: io.github.theforceengine.tfe.appdata.xml
+      # Add a wrapper script that points TFE_DATA_HOME to the correct location within the sandbox
+      - type: script
+        dest-filename: io.github.theforceengine.tfe.sh
+        commands:
+          - # Check for game data
+          - if [[ -z "$TFE_SKIP_DATACHECK" ]] && [[ -z $(find "$XDG_DATA_HOME" -iname dark.gob) ]] && [[ ! -d "$HOME/.local/share/Steam/steamapps/common/Dark Forces/Game/" ]]; then
+          -     zenity --error --text "<b>Could not find Dark Forces game data</b>\n\n
+                    Please either install Dark Forces via Steam to your library in <tt><b>$HOME/.local/share/Steam/</b></tt>
+                    or copy the game data to <tt><b>$XDG_DATA_HOME/</b></tt>."
+                    --ok-label "Close" --width=400
+          - fi
+          - export TFE_DATA_HOME=/app/share/TheForceEngine
+          - exec theforceengine
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DDISABLE_SYSMIDI=ON
+    builddir: true
+    build-commands:
+      # Install launcher script
+      - install -Dm 744 ../io.github.theforceengine.tfe.sh -t /app/bin
+      # Edit and install desktop launcher file
+      - desktop-file-edit --set-icon=io.github.theforceengine.tfe ../TheForceEngine/TheForceEngine.desktop
+      - desktop-file-edit --set-key=Exec --set-value=io.github.theforceengine.tfe.sh ../TheForceEngine/TheForceEngine.desktop
+      - install -Dm 644 ../TheForceEngine/TheForceEngine.desktop /app/share/applications/io.github.theforceengine.tfe.desktop
+      # Install icon
+      - install -Dm 644 ../TheForceEngine/TheForceEngine.png /app/share/icons/hicolor/256x256/apps/io.github.theforceengine.tfe.png
```

Comment:
This is unnecessary:
```
-- Installing: /app/share/icons/hicolor/256x256/apps/TheForceEngine.png
```

So is line 63. You can use `rename-icon` and `rename-desktop-file` to rename them. `rename-icon` also make line 61 unecessary.